src/Compilers/CSharp/Portable/Syntax/SyntaxNormalizer.cs

```diff
@@ -425,6 +425,10 @@ private static int LineBreaksAfterSemicolon(SyntaxToken currentToken, SyntaxToke
             {
                 return 0;
             }
+            else if (currentToken.Parent.IsKind(SyntaxKind.PropertyDeclaration))
+            {
+                return 2;
```

Comment:
i'm a tad bit confused by this.  can you give an example of a property here and what currentToken refers to in it?

Reply:
Lets take a look at
```cs
class C
{
    public string P { get; set; } = "xyz";
}
```
Here there are 3 `;`s in the property. `;`s of `get` and `set` have a parent of `Get(Set)AccessorDeclaration`. The ending `;` is a direct child of a property. Yes, it was a bit surprising for me as well, but the `;` is not part of `= "xyz"`. You can take close look at parse tree at [sharplab](https://sharplab.io/#v2:EYLgtghglgdgNAFxBAzggPgAQEwAIDCAsAFADeJulumAzNQIwAMuACrqbgOYCmCA3LhS8BAX1wBeXACIAHgE8AXlL4kRQA==).
So the above conditions applies to `;` at the end of a property and adds 2 line breaks there.

Reply:
So, are we not consistent between properties with initializers and properties without?

Reply:
@Youssef1313 Properties without initializers won't even step into this method at that stage. This method is only handling cases after semicolon. If a properry doesn't have an initializer, line breaks are computed around closing brace and that path is already correct

Reply:
Consider adding an assertion that this is the initializer's semi-colon. This is a bit stronger than a comment.

Reply:
> There can be no initializer when the semicolon is attached to the property

I'm not sure what this means.  the semicolon doesn't belong to the initializer, it belongs to the property.  Maybe i'm not quite understanding what you mean here.

Reply:
nit: The assertion I'm proposing would be along the lines of `((PropertySyntax)currentToken.Parent).Semicolon == currentToken`.